hack/prebuild/cluster_setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export TF_VAR_aws_user="test-user"
```

Comment:
Is this the same as `AWS_PROFILE`?

Reply:
No, because the default one is "default". That should be the AWS username.

Reply:
Like
```bash
aws iam get-user | jq --raw-output '.User.UserName'
```
?

Reply:
exactly

Reply:
So use it :smile: